generators/typescript/model/type-reference-example-generator/src/GeneratedTypeReferenceExampleImpl.ts

```diff
@@ -42,6 +42,11 @@ export class GeneratedTypeReferenceExampleImpl implements GeneratedTypeReference
                     integer: (integerExample) => ts.factory.createNumericLiteral(integerExample),
                     double: (doubleExample) => ts.factory.createNumericLiteral(doubleExample),
                     long: (longExample) => ts.factory.createNumericLiteral(longExample),
+                    uint: (uintExample) => ts.factory.createNumericLiteral(uintExample),
+                    uint64: (uint64Example) => ts.factory.createNumericLiteral(uint64Example),
+                    float: (floatExample) => ts.factory.createNumericLiteral(floatExample),
+                    bigInteger: (bigIntegerExample) => ts.factory.createNumericLiteral(bigIntegerExample),
```

Comment:
We'll need big integers to actually be typed as `string` in TypeScript.

```suggestion
                    bigInteger: (bigIntegerExample) => ts.factory.createStringLiteral(bigIntegerExample),
```